### PR TITLE
Removed the empty spaces and resolved the build issue.

### PR DESCRIPTION
**Pull request by Venkadesan0210**
Additionally,
1) Solved the SonarQube issue.
2)Adding the application.properties for the purpose of database connection.
Final_SonarQube_Report

### CODE REVIEW

File: LeaveManagement/LeaveManagement/src/test/java/com/example/LeaveManagement/LeaveManagementApplicationTests.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package com.example.leavemanagement;
+import org.springframework.boot.test.context.SpringBootTest;
+@SpringBootTest
+class LeaveManagementApplicationTests {
```
Comment (shyamsundarv1489) [inline LeaveManagement/LeaveManagement/src/test/java/com/example/LeaveManagement/LeaveManagementApplicationTests.java:4]: Ensure the unit test coverage is addressed.

File: LeaveManagement/LeaveManagement/src/main/java/com/example/LeaveManagement/service/JwtService.java [hunk 2]
```diff
@@ -70,19 +70,38 @@ public UserDetails loadUserByUsername(String username) throws UsernameNotFoundEx
...
             authenticationManager.authenticate(new UsernamePasswordAuthenticationToken(userName, userPassword));
         } catch (DisabledException e) {
-            throw new Exception("USER_DISABLED", e);
+            throw new UserDisabledException("USER_DISABLED", e);
         } catch (BadCredentialsException e) {
-            throw new Exception("INVALID_CREDENTIALS", e);
+            throw new InvalidCredentialsException("INVALID_CREDENTIALS", e);
+        }
+    }
+
```
Comment (shyamsundarv1489) [inline LeaveManagement/LeaveManagement/src/main/java/com/example/LeaveManagement/service/JwtService.java:87]: Remove empty lines

File: LeaveManagement/LeaveManagement/src/test/java/com/example/LeaveManagement/LeaveManagementApplicationTests.java (hunk 1 above)
Comment (Venkadesan0210) [reply LeaveManagement/LeaveManagement/src/test/java/com/example/LeaveManagement/LeaveManagementApplicationTests.java:4]: Addressing the PR comments provided above, I have planned to improve the unit test coverage.

File: LeaveManagement/LeaveManagement/src/main/java/com/example/LeaveManagement/service/JwtService.java (hunk 2 above)
Comment (Venkadesan0210) [reply LeaveManagement/LeaveManagement/src/main/java/com/example/LeaveManagement/service/JwtService.java:87]: Addressing the PR comments provided above has been resolved.

**Pull request merged**